cypress/support/fragments/marcAuthority/marcAuthorities.js

```diff
@@ -779,6 +779,17 @@ export default {
       );
     }
   },
+
+  selectByTypeAndHeading(heading, type) {
```

Comment:
This method doesn't do what you expect. Just select authority by title. Such method already exists.
Because you choose record with uniqe title, you can just use selectItem() method